analytics/src/docker/Dockerfile

```diff
@@ -6,6 +6,9 @@ RUN java -jar "$JETTY_HOME/start.jar" --add-to-startd=jmx,jmx-remote,stats
 
 ADD . /
 
+RUN mkdir -p /etc/georchestra && \
+    chown jetty:jetty /etc/georchestra
+
 VOLUME [ "/tmp", "/run/jetty" ]
```

Comment:
Because I want to leave the choice to people running the image to have the config in `/etc/georchestra` as an external volume or to create self-contained images (app+config)